ui/scss/component/_form-field.scss

```diff
@@ -249,10 +249,8 @@ input-submit {
   label::after {
     height: 6px;
     width: 12px;
-    border-left: 2px solid;
-    border-bottom: 2px solid;
-    border-color: var(--color-primary);
-    border-left-color: var(--color-primary);
+    border-left: 2px solid var(--color-primary);
+    border-bottom: 2px solid var(--color-primary);
```

Comment:
Correct me if I'm wrong, but I believe this is the intended style.

Reply:
@toshokanneko can you confirm?

Reply:
Just double checking here to see if all is well on this front.

Reply:
@tzarebczan @toshokanneko Just wanted to check back on this one again.

Reply:
Looks good to me